src/Mvc/src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/DefaultJsonHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Html;
+
+namespace Microsoft.AspNetCore.Mvc.Rendering
+{
+    internal class DefaultJsonHelper : IJsonHelper
+    {
+        public IHtmlContent Serialize(object value)
+        {
+            throw new NotSupportedException();
+        }
```

Comment:
?????

Reply:
Same deal as DefaultTempDataSerializer

Reply:
Tracking implementing this via https://github.com/aspnet/AspNetCore/issues/6202